worker/src/index.ts

```diff
@@ -19,33 +20,42 @@ interface ErrorResult<T> {
   error: T;
 }
 
-interface RequestSettings {
+export interface RequestSettings {
   stream: boolean;
   ff_stream_force_format?: boolean;
   ff_increase_timeout?: boolean;
 }
 
 export type Result<T, K> = SuccessResult<T> | ErrorResult<K>;
 
-function forwardRequestToOpenAi(
+export async function forwardRequestToOpenAi(
   request: Request,
   requestSettings: RequestSettings,
-  body?: string
+  body?: string,
+  retryOptions?: RetryOptions,
 ): Promise<Response> {
   let url = new URL(request.url);
   const new_url = new URL(`https://api.openai.com${url.pathname}`);
   const headers = removeHeliconeHeaders(request.headers);
   const method = request.method;
   const baseInit = { method, headers };
   const init = method === "GET" ? { ...baseInit } : { ...baseInit, body };
+
+  let response;
```

Comment:
hot take, I hate mutable variables. I would rather add this check to the call or wrap it. 

It's a super nit picky thing, but it can reduce bugs in the future

Reply:
not block at all, let's merge this in